base/float.jl

```diff
@@ -844,8 +844,10 @@ end
 """
     floatmin(T = Float64)
 
-Return the smallest positive normal number representable by the floating-point
-type `T`.
+Return the smallest positive floating-point number (which is not sub-normal)
```

Comment:
This seems wrong: A parenthetical remark like this could be read as a descriptive clause, whereas what we want is a [restrictive clause](https://www.grammarly.com/blog/using-that-and-which-is-all-about-restrictive-and-non-restrictive-clauses/).

The original phrasing seems better to me.

Reply:
So it shouldn't be in bracket or what do you suggest?

Reply:
Or just "Return the smallest positive non-subnormal number of floating-point type T". Does the word "representable" add anything?

Reply:
I think the word "representable" does, since not all numbers can be represented exactly by a type. I would go with @stevengj on this.

Reply:
The object returned *is* of that type. What does it mean in addition to be *representable* by it, that's my question?

Reply:
I think you have a point here, since there can't be any overflow or stuffs like that....Hmm🤔.

I think the "representable" here just seems to distinguish the fact that the results are different for different floating-point types, since one would assume it will be same for all floating point types, since they're all "floating-points".

An example:
```julia
julia> x = floatmin()
2.2250738585072014e-308

julia> Float32(x)
0.0f0

julia> Float64(x)
2.2250738585072014e-308
```

So yes, in my view the word "representable" is valid there, just to add some thought to it.

Reply:
Right, it's not invalid. I guess it depends whether you think it's returning a mathematical number (which tells you about some floating point behaviour, but could be written down on a post-it note) or returning a computer object "of floating-point type T" (which then simply exemplifies the property).

Since it does always return a `T`, my complaint is that adding more words makes it sound like there is more to un-wrap here. Like you need to understand both what the object is and what it represents, are these different concepts? My take is that they are not.